packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-processor.spec.ts

```diff
@@ -16,8 +17,9 @@ describe('AmazonPayV2PaymentProcessor', () => {
     let amazonPayV2ScriptLoader: AmazonPayV2ScriptLoader;
     let processor: AmazonPayV2PaymentProcessor;
     let amazonPayV2SDKMock: AmazonPayV2SDK;
+    let amazonPayV2Mock: PaymentMethod
 
-    beforeEach(() => {
+    beforeEach(async () => {
```

Comment:
Does it make sense to initialise the processor in the beforeEach block? Or can we initialise it in the `it` blocks itself?

Reply:
It can be both ways, but I've done it on each test closure for clarity.